README.md

```diff
@@ -0,0 +1,18 @@
+# cloud-provider-sample
+
+This is a sample of how to build a cloud provider repo. This will build a Kubernetes image which deploys on bare metal.
```

Comment:
This is the intent. I think it would be good to warn people its a WIP until its actually working.

Reply:
Oh good point! I have added a note. :+1: